ql/ql/src/codeql_ql/ast/Ast.qll

```diff
@@ -186,7 +186,9 @@ class QueryDoc extends QLDoc {
   override string getAPrimaryQlClass() { result = "QueryDoc" }
 
   /** Gets the @kind for the query */
-  string getQueryKind() { result = this.getContents().regexpCapture("(?s).*@kind (\\w+)\\s.*", 1) }
+  string getQueryKind() {
+    result = this.getContents().regexpCapture("(?s).*@kind ([\\w-]+)\\s.*", 1)
```

Comment:
For maximal future-proofing, we could also just match a non-zero number of _non_-whitespace characters instead.

Reply:
I think I prefer to keep it as is.  
E.g. I'm not sure `@` is valid in an alert-message.